packages/app-elements/src/ui/forms/InputSelect/AsyncComponent.tsx

```diff
@@ -53,7 +54,11 @@ export const AsyncSelectComponent: React.FC<AsyncSelectComponentProps> = ({
       {...rest}
       defaultOptions={initialValues}
       onChange={onSelect}
-      noOptionsMessage={() => noOptionsMessage}
+      noOptionsMessage={({ inputValue }) =>
+        isEmpty(inputValue) && initialValues.length === 0
+          ? 'Type to search'
```

Comment:
I would add an option with default value of `Type to search`, so that it would be possible to also customize this message, something like `placeholderMessage`

Reply:
Alright! I just added a dedicated prop in Async component..